frontend/assets/stylesheets/components/_payment-b.scss

```diff
@@ -0,0 +1,261 @@
+    }
+
+    .u-cf {
+        label {
+            width: 50%;
+
+            @include mq($from: tablet) {
+                width: 49%;
+            }
+
+            float: left;
+        }
+
+        div {
+            width: 50%;
+
+            @include mq($from: tablet) {
+                width: 51%;
+            }
+
+            float: right;
+
+            @include mq($from: tablet) {
+                margin-left: 0;
+            }
+
+            ul > li:nth-child(1) {
+                text-align: left;
+            }
+
+            br {
+                @include mq($from: tablet) {
+                    display:none;
+                }
+            }
+        }
+    }
+
+    .credit-card-bottom-description {
+        @include fs-textSans(2);
+        p {
+            margin-bottom: 0;
+        }
+
+        div {
+            display: block;
+
+            @include mq($from: tablet) {
+                display: none;
+            }
+        }
+    }
+
+    .credit-card-note {
+        display: none;
+
+        @include mq($from: tablet) {
+            display: block;
+        }
+    }
+
+    .promo-code {
+        button {
+            background: transparent;
+            color: #005689;
+            border-color: #005689;
+            background: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30"><path fill="#005689" d="M22.8 14.6L15.2 7l-.7.7 5.5 6.6H6v1.5h14l-5.5 6.6.7.7 7.6-7.6v-.9"/></svg>') center right no-repeat;
+            padding-right: 10px;
+        }
+
+        button::after {
+            background: none;
+        }
+    }
+
+    .form-section__bottom-progress-bar {
+        @include mq($from: tablet) {
+            display: none;
+        }
+    }
+
+    .actions {
+        button {
+            @include fs-textSans(3);
+            height: 50px;
+            border-radius: 650px;
+            padding-left: 20px;
+            padding-right: 55px;
+            background: var(--c-form-button--submit) inline("arrow-white-right-circle.svg") center right no-repeat;
+            background-position: right 0.7rem center;
+            font-size: 20px;
+            font-weight: normal;
+            padding-top: 12px;
+            background: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="36" height="36" viewBox="-9 -9 36 36"><path fill="#FFF" d="M0 9.343h14.952L9.09 16.36l.76.76L18 8.95v-.78L9.85 0l-.76.762 5.862 7.016H0"/><path fill="#FFF" d="M9 27C-.924 27-9 18.925-9 9S-.925-9 9-9C18.927-9 27-.925 27 9S18.926 27 9 27zM9-8.056C-.403-8.056-8.055-.406-8.055 9c0 9.405 7.652 17.057 17.057 17.057 9.405 0 17.057-7.65 17.057-17.057C26.057-.405 18.405-8.056 9-8.056z" opacity=".3"/></svg>') center right no-repeat;
+            background-position: right 0.7rem center;
+            background-color : guss-colour(guardian-brand);
+            border-color : guss-colour(guardian-brand);
+
+
+            &:hover {
+                background-color: #003557;
+                border-color: #003557;
```

Comment:
This colour doesn't belong to the Guardian Palette https://github.com/guardian/pasteup-palette/blob/master/src/_palette.scss